thirdeye/thirdeye-frontend/app/pods/manage/alert/tune/route.js

```diff
@@ -317,6 +318,14 @@ export default Route.extend({
       timeRangeOptions: setUpTimeRangeOptions([durationDefault], duration)
     });
     controller.initialize();
+
+    // Ensure date range picker gets populated correctly
+    later(this, function() {
+      controller.setProperties({
```

Comment:
let's not mix old and new module apis. Good job on using `later` here, let's also use `get(this,...)` and `setProperties(controller, ....`